### PR TITLE
fix: prevent splitting on an undefined value

### PR DESCRIPTION
**Pull request by saemik94**
## Summary
Fixed a bug where the imageList.widget crashed, due to empty digest or tag on docker images.

## Proposed Changes

  -

## Checklist

- [ ] I added tests
- [x] I updated the README if necessary
- [ ] This PR introduces a breaking change
- [x] Fixed issue #246
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

**Comment by saemik94** [comment]
I would prefer to use the concatenation operator to check the values, but the linter prevents the use of this operator.

`image.RepoDigests && image.RepoDigests.length` vs. `image.RepoDigests?.length`

The dependency "default": "^12.0.1" would need to be updated, the current version is 17.1.0 and the operator is supported from version 15. But this leads to a whole list of adjustments that would then have to be made. I am not sure how to handle this?

File: widgets/images/imageList.widget.js [hunk 1]
```diff
@@ -45,11 +45,11 @@ class myWidget extends ListWidget {
 
     if (images) {
       images.forEach((image) => {
-        const getTag = (tag, part) => tag ? tag[0].split(':')[part] : 'none'
+        const getTag = (tag, part) => (tag && tag.length) ? tag[0].split(':')[part] : 'none'
 
         imageList.push([
           image.Id.substring(7, 12),
-          image.RepoDigests ? image.RepoDigests[0].split('@')[0] : getTag(image[2], 0),
+          (image.RepoDigests && image.RepoDigests.length) ? image.RepoDigests[0].split('@')[0] : getTag(image[2], 0),
```
Comment (saemik94) [inline widgets/images/imageList.widget.js:52]: I would prefer to use the concatenation operator to check the values, but the linter prevents the use of this operator.

`image.RepoDigests && image.RepoDigests.length` vs. `image.RepoDigests?.length`

The dependency "default": "^12.0.1" would need to be updated, the current version is 17.1.0 and the operator is supported from version 15. But this leads to a whole list of adjustments that would then have to be made. I am not sure how to handle this?

Comment (lirantal) [reply widgets/images/imageList.widget.js:52]: It's ok to use as-is and skip the optional chaining due to support of older Node.js versions.

**Review by lirantal — APPROVED** [review]

**Comment by lirantal** [comment]
@saemik94 thanks, I'll land this change and it will follow with publishing a new npm version and a new docker image tag to Docker Hub.

Thank you so much! ❤️

**Pull request merged**